api/v1alpha1/lvmcluster_types.go

```diff
@@ -90,6 +90,10 @@ type DeviceSelector struct {
 	// MinSize is the minimum size of the device which needs to be included. Defaults to `1Gi` if empty
 	// +optional
 	// MinSize *resource.Quantity `json:"minSize,omitempty"`
+
+	// Paths are the complete location of block devices, which needs to be included while creating a VG.
+	// +optional
+	Paths []string `json:"paths,omitempty"`
```

Comment:
`DevicePaths` should be a better name than `Paths`, IMO.

Reply:
I see now that's this field is already inside `DeviceSelector`.  So `Paths` is also fine.

Reply:
keeping it as paths for now